README.md

```diff
@@ -24,7 +24,32 @@ This service can be used with any project type. The examples below are Drupal-sp
   - Drupal Test Traits
     - Ensure you have a working site that has the `weitzman/drupal-test-traits` Composer package.
     - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit --bootstrap=../vendor/weitzman/drupal-test-traits/src/bootstrap-fast.php --printer '\Drupal\Tests\Listeners\HtmlOutputPrinter' ../vendor/weitzman/drupal-test-traits/tests/ExampleSelenium2DriverTest.php"`
-- On your host, browse to https://[DDEV SITE URL]:7900 (password: `secret`) to watch tests run (neat!).
+
+## Watching the tests
+
+### The easy way: Use noVNC (built-in)
+
+On your host, browse to https://[DDEV SITE URL]:7900 (password: `secret`) to watch tests run with noVNC (neat!).
+
+This is a no-configuration solution that enables you to quickly see what is going on with your tests.
```

Comment:
Considering we need to make configuration changes and restart DDEV, I'm not so sure about the wording here. 🤷

Reply:
Yeah, I didn't realize that `--headless` is set in the YAML file, so there should probably be a section on how and why to disable headless as well.  What's the reason for setting `--headless` as a default in the first place?  Is it just speed?

Reply:
I imaging there would be memory savings from using `--headless`, which might be more important when running on CI.

Note there are 2 references, one for Mink tests and one forr DTT tests.

@weitzman , are there issues removing this as the default?